exercises/practice/two-fer/two_fer.vim

```diff
@@ -10,5 +10,5 @@
 "" One for Alice, one for me.
 ""
 function! TwoFer(name) abort
-  "your code goes here"
+  " your code goes here"
```

Comment:
Why a space here?

Reply:
Also, we could probably set it as the example, `"One for Alice, one for me."`

Reply:
There's a space between the double quotes and the message to implement the function for the other exercises I checked so I added it for consistency.

Reply:
Doh.  Yep, comment marker, not string delimiter.

Reply:
Should we use the closing quotes then?

Reply:
Yeah, just double checked, it is not terminated with another comment marker for at least the first few exercises.